src/app/backend/handler/apihandler.go

```diff
@@ -2143,6 +2143,7 @@ func (apiHandler *APIHandler) handleLogs(request *restful.Request, response *res
 	if err != nil {
 		refLineNum = 0
 	}
+	previous := request.QueryParameter("previous") == "true"
```

Comment:
Could you explain meaning of `previous` flag more?

Reply:
I hestitated to write a long comment in the apihandler. Instead, I made the variable a bit more verbose and added comment in handler function.